spec/requests/catalog_request_spec.rb

```diff
@@ -44,6 +44,19 @@
     end
   end
 
+  context "when requesting a URL from the legacy system" do
+    it "redirects requests for '/collections/eadid/componentid' to '/catalog/eadid_componentid'" do
+      get "/collections/C0140/c03411"
+      expect(response.code).to eq "301"
+      expect(response).to redirect_to("http://www.example.com/catalog/C0140_c03411")
+    end
+    it "redirects requests correctly when there are dots in the eadid" do
+      get "/collections/MC001.02.01/c00003"
+      expect(response.code).to eq "301"
+      expect(response).to redirect_to("http://www.example.com/catalog/MC001.02.01_c00003")
```

Comment:
I'm real sorry to do this to you, and it's probably going to complicate things, but the actual URL is https://findingaids-beta.princeton.edu/catalog/MC001-02-01_c00003. I should have remembered, my bad.